### PR TITLE
[asciidoc/en] fixes #2644 add `Further Reading` section to AsciiDoc

### LINKED ISSUE #2644: [asciidoc/en]
Consider adding implementations reference for `asciidoctor` (http://asciidoctor.org/).
The current page offers a good intro into the language but does not offer any guide to users wanting to test it out.

Right now there are two options:
· AsciiDoc: original python lib. Not sure about it's current state. Github seems active (https://github.com/asciidoc/asciidoc), but the site does not list a release since 2013 (http://www.methods.co.nz/asciidoc/CHANGELOG.html).
· Asciidoctor: Ruby alternative. Pretty active a with integration with Java and JavaScript.

### PR DESCRIPTION
**Pull request by abelsromero**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [ ] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [ ] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by abelsromero** [comment]
I swear, I tried not to be biased toward Asciidoctor.

Btw, if everything is fine for you, I can add Spanish and Catalan versions also. Let me know if you want them in this PR or another.

File: asciidoc.html.markdown [hunk 1]
```diff
@@ -120,3 +121,16 @@ You can nest lists by adding extra asterisks or periods up to five times.
 .... foo 4
 ..... foo 5
 ```
+
+## Further Reading
+
+There are two tools to process AsciiDoc documents:
+
+1. [AsciiDoc](http://asciidoc.org/): original Python implementation available in the main Linux distributions. Stable and currently in maintenance mode.
+2. [Asciidoctor](http://asciidoctor.org/): alternative Ruby implementation, usable also from Java and JavasScript. Under active development, it aims to extend the AsciiDoc syntax with new features and output formats.
```
Comment (Oire) [inline asciidoc.html.markdown]: JavaScript?

File: asciidoc.html.markdown [hunk 2]
```diff
@@ -120,3 +121,16 @@ You can nest lists by adding extra asterisks or periods up to five times.
...
+
+Following links are related to `Asciidoctor` implementation:
+
+* [Markdown - AsciiDoc syntax comparision](http://asciidoctor.org/docs/user-manual/#comparison-by-example): side-by-side comparision of common Markdown and AsciiDoc elements.
```
Comment (Oire) [inline asciidoc.html.markdown]: Comparison (two times)

**Comment by abelsromero** [comment]
done! I'll send another PR with Es and Ca, late this week

**Comment by Oire** [comment]
Thanks a lot!

**Pull request merged**